static/styles/zulip.scss

```diff
@@ -390,6 +390,7 @@ li,
     left: -5px;
     margin-top: -24px;
     display: block;
+    z-index: 2;
```

Comment:
Can you add a comment repeating the commit message explanation of why this is needed?  Seems like valuable context for posterity.

Reply:
Adding opacity of 0.5 to non-editable text made the button unclickable. Increasing it's z-index makes it clickable again.

Reply:
Oh you meant a comment in the CSS file😅

Reply:
Yep!